src/pydocstyle/checker.py

```diff
@@ -680,6 +681,10 @@ def check(filenames, select=None, ignore=None, ignore_decorators=None):
         try:
             with tokenize_open(filename) as file:
                 source = file.read()
+                try:
+                    source = source.decode('utf-8')
+                except:
```

Comment:
To catch any exception use the ```except Exception```, instead of ```except BaseException``` or just ```except```.
Indicate the most specific exception type in ```except``` block.
For example:
```python
# Bad
try:
    mapping[key]
except Exception:
    ...

# Better
try:
    mapping[key]
except KeyError:
    ...
```